src/components/ec-table/ec-table.vue

```diff
@@ -35,7 +35,11 @@
               class="ec-table__cell"
               :class="[
                 getStickyColumnClass(colIndex, columns),
-                { 'ec-table__cell--text-center': columns[colIndex] && columns[colIndex].type === 'icon' }]"
+                {
+                  'ec-table__cell--is-type-icon': columns[colIndex] && columns[colIndex].type === 'icon',
+                  'ec-table__cell--is-type-currency': columns[colIndex] && columns[colIndex].type === 'currency',
```

Comment:
@jafcalvente I'm thinking "currency" is too specific, maybe would it make more sense amount type? What do you think?

Reply:
The name of `currency` for the type was decided in a previous discussion of this same PR, and was the indicated one in the conversation that we had with UX by Slack.
A couple of reasons because I prefer to keep it:

- It is true that `currency` may seem generic (since the cell shows Symbol + Amount), but by calling it only `amount`, we are in the same generalization. If at some point we wanted to have a type that referred only to amount, we couldn't. However, if we only wanted to see the symbol we still could have the type `symbol`.

- On the other hand, the Chameleon component that allows us to enter symbol and amount is already called `Currency Input`, so for the sake of homogeneity, it would be good to keep the same pattern.